packages/plugin-ext/src/api/plugin-api.ts

```diff
@@ -474,7 +474,10 @@ export interface StatusBarExt {
 export enum EditorPosition {
     ONE = 0,
     TWO = 1,
-    THREE = 2
+    THREE = 2,
+    FOUR = 3,
+    FIVE = 4,
+    SIX = 5
```

Comment:
Should it be aligned with vscode view position? It goes there up to 9.

Reply:
fixed